src/include/daos/object.h

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright 2016-2018 Intel Corporation.
+ * (C) Copyright 2016-2019 Intel Corporation.
```

Comment:
Oops?

Reply:
Came in with a merge I guess? I can kick them out.

Reply:
Hmm. I think my pre-commit hook that is checking for copyright headers/dates noticed these files due to the merge?